tensilelite/Tensile/Common.py

```diff
@@ -1220,7 +1223,9 @@ def getArchitectureName(gfxName):
     "Activation":               False,
     "ActivationNoGuard":        False,
     # For kernels putting arguments in workspaces instead of kernel arguments, they can choose to support user arguments input instead.
-    "SupportUserArgs":          True
+    "SupportUserArgs":          True,
+
+    "BiasDim":                  False
```

Comment:
It is better to expand the scope of UseBias:
My suggestion is 
UseBias = 0 : No Bias
UseBias = 0x1:  support BiasDim M direction only
UseBias = 0x2: support BiasDim N direction only
UseBias = 0x3: support BiasDim M/N both

Reply:
Use Parameter "UseBias" to enable the bias on M/N directions
 *0=No bias, 1=Bias at  M direction, 2=Bias at  N direction, 3=Bias at M
or N direction
 *tag in kernel name, 1=*_Bias_*, 2=*_Bias_BDN_*, 3=*_Bias_BDMN_*
 
 new code is at here
 https://github.com/ROCm/hipBLASLt/pull/566/commits/be56fdad4516558712b565e75fba39ba92b15f05#